Sources/FoundationEssentials/Data/Data.swift

```diff
@@ -2198,7 +2202,103 @@ public struct Data : Equatable, Hashable, RandomAccessCollection, MutableCollect
     public func withUnsafeBytes<ResultType>(_ body: (UnsafeRawBufferPointer) throws -> ResultType) rethrows -> ResultType {
         return try _representation.withUnsafeBytes(body)
     }
-    
+
+    @available(FoundationSpan 6.2, *)
+    public var bytes: RawSpan {
+        @lifetime(borrow self)
+        borrowing get {
+            let buffer: UnsafeRawBufferPointer
+            switch _representation {
+            case .empty:
+                buffer = UnsafeRawBufferPointer(start: nil, count: 0)
+            case .inline:
+                buffer = unsafe UnsafeRawBufferPointer(
+                  start: UnsafeRawPointer(Builtin.addressOfBorrow(self)),
```

Comment:
Are we opening the door here for Foundation to use the `Builtin` module generally? Or is this a placeholder for some stdlib API/SPI cover that's coming down the road?